Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H

```diff
@@ -102,86 +105,87 @@ void UpdateMomentumPerezElastic (
+                        (T_PR(2.0)*MathConst::pi*PhysConst::ep0*muRst*vrelst*vrelst_invar);
+
 
         // Compute the Coulomb log lnLmd first
         T_PR lnLmd;
         if ( L > T_PR(0.0) ) { lnLmd = L; }
         else
         {
-            // Compute b0 according to eq (22) from Perez et al., Phys.Plasmas.19.083104 (2012)
-            // Note: there is a typo in the equation, the last square is incorrect!
-            // See the SMILEI documentation: https://smileipic.github.io/Smilei/Understand/collisions.html
-            // and https://github.com/ECP-WarpX/WarpX/files/3799803/main.pdf from GitHub #429
-            T_PR const b0 = amrex::Math::abs(q1*q2) * inv_c2 /
-                (T_PR(4.0)*MathConst::pi*PhysConst::ep0) * gc/mass_g *
-                ( m1*g1s*m2*g2s/(p1sm*p1sm*inv_c2) + T_PR(1.0) );
-
-            // Compute the minimal impact parameter
-            constexpr T_PR hbar_pi = static_cast<T_PR>(PhysConst::hbar*MathConst::pi);
-            const T_PR bmin = amrex::max(hbar_pi/p1sm, b0);
+
+            // Compute the minimum impact parameter from quantum: bqm = lDB/(4*pi) = hbar/(2*p1sm)
+            // See NRL formulary. Also see "An introduction to the physics of the Coulomb logarithm,
+            // with emphasis on quantum-mechanical effects", J. Plasma Phys. vol. 85 (2019). by J.A. Krommes.
+            // Note: The formula in Perez 2012 and in Lee and More 1984 uses h rather than
+            // hbar for bqm. If this is used, then the transition energy where bmin goes from classical
+            // to quantum is only 2.5 eV for electrons; compared to 100 eV when using hbar.
+            const T_PR bmin_qm = static_cast<T_PR>(PhysConst::hbar*0.5/p1sm);
+
+            // Set the minimum impact parameter
+            const T_PR bmin = amrex::max(bmin_qm, T_PR(0.5)*b0);
 
             // Compute the Coulomb log lnLmd
             lnLmd = amrex::max( T_PR(2.0),
-                    T_PR(0.5)*std::log(T_PR(1.0)+lmdD*lmdD/(bmin*bmin)) );
+                    T_PR(0.5)*std::log(T_PR(1.0) + bmax*bmax/(bmin*bmin)) );
         }
 
-        // Compute s
-        const auto tts = m1*g1s*m2*g2s/(inv_c2*p1sm*p1sm) + T_PR(1.0);
-        const auto tts2 = tts*tts;
-        s = n1*n2/n12 * dt*lnLmd*q1*q1*q2*q2 /
-            ( T_PR(4.0) * MathConst::pi * PhysConst::ep0 * PhysConst::ep0 *
-                m1*g1*m2*g2/(inv_c2*inv_c2) ) * gc*p1sm/mass_g * tts2;
-
-        // Compute s'
-        const auto cbrt_n1 = std::cbrt(n1);
-        const auto cbrt_n2 = std::cbrt(n2);
-        const auto coeff = static_cast<T_PR>(
-            std::pow(4.0*MathConst::pi/3.0,1.0/3.0));
-        T_PR const vrel = mass_g*p1sm/(m1*g1s*m2*g2s*gc);
-        T_PR const sp = coeff * n1*n2/n12 * dt * vrel * (m1+m2) /
-            amrex::max( m1*cbrt_n1*cbrt_n1,
-                        m2*cbrt_n2*cbrt_n2);
-
-        // Determine s
-        s = amrex::min(s,sp);
+        // Compute s12 with sigma limited by sigma_max where mfp = atomic spacing
```

Comment:
Could add a reference in the comment? (Is it the same reference as above?)

Reply:
Using sigma_max is the same as using sp from Perez 2012. Did you get a chance to look at the tech-note I sent you and Axel where I went over the different limiting models used by Perez?

Reply:
This is great. If you like, the PDF of the proof linked and the tech note could also be part of our [theory sections](https://warpx.readthedocs.io/en/latest/theory/intro.html). (Relatively quick to translate from LaTeX to `.rst`.)